phpBB/styles/prosilver/theme/common.css

```diff
@@ -603,6 +604,53 @@ a.header-avatar span:after {
 	display: none !important;
 }
 
+/* Smiley Dropdown---------------------------------------- */
+.smiley-box {
+	float: right;
+	width: 100%;
+}
+
+.smiley-box .smiley-image {
+	margin: 3px;
+}
+
+.dropdown .smiley-content {
+	width: 200px;
+}
+
+.smiley-box .smiley-list-item {
+	border: none;
+	display: inline-block;
+}
+
+.smiley-list-item .smiley-add {
+	padding: 0;
+}
+
+.button .dropdown-left .dropdown {
+	right: -12px;
+}
+
+.button .dropdown-right .dropdown {
+	left: -12px;
+}
+
```

Comment:
also the footer still needs some work the border should span all the way across? the padding should be applied internally rather than on the wrap?